src/components/panel/panel.spec.js

```diff
@@ -868,6 +895,76 @@ describe('$mdPanel', function() {
       });
     });
 
+    describe('CSS class logic:', function() {
+      it('should add a class to the container/wrapper when toElement=false',
+        function() {
+        openPanel(DEFAULT_CONFIG);
+
+        panelRef.addClass('my-class');
+
+        expect(PANEL_WRAPPER_CLASS).toHaveClass('my-class');
```

Comment:
Let's add another expect here to be safe. Technically, the class can be add to the wrapper AND element and still pass all of these tests. Same with the below tests.

```
expect(PANEL_WRAPPER_CLASS).toHaveClass('my-class');'
expect(PANEL_EL).not.toHaveClass('my-class');
```